### PR TITLE
Removed flypme altcoin integration and updated checkout payment screen

### PR DESCRIPTION
**Pull request by cnohall**
- Removed flypme altcoin integration, both in the PHP and angular files.
- Updated the checkout payment screen to look like the checkout payment screen for the woocommerce plugin.

### CODE REVIEW

File: css/order.css [hunk 1]
```diff
@@ -69,53 +69,59 @@
...
-  text-align: center;
+.bnomics-grey {
+  color:grey;
 }
 
 .bnomics-address {
+  margin-top: 10px;
   position:relative;
   margin:auto !important;
   width: 90% !important;
```
Comment (DarrenWestwood) [inline css/order.css:80]: The 90% width restraint can be removed as it is causing the address and amount inputs to vary in lengths.
EDD Spacing

File: order.php [hunk 2]
```diff
@@ -33,67 +25,52 @@
...
+                  <a href="bitcoin:{{order.address}}?amount={{order.satoshi/1.0e8}}" target="_blank"><?=__('Open in wallet', 'edd-blockonomics')?></a>
+                </div>
               </div>
               <!-- BTC Amount -->
               <div class="bnomics-amount">
               <div class="bnomics-bg">
                 <!-- Order Status -->
                 <div class="bnomics-order-status-wrapper">
-                  <span class="bnomics-order-status-title" ng-show="order.status == -1" ng-cloak ><?=__('To confirm your order, please send the exact amount of <strong>BTC</strong> to the given address', 'edd-blockonomics')?></span>
+                  <span class="bnomics-order-status-title" ng-show="order.status == -1" ng-cloak ><?=__('To pay, send exactly this BTC {{altcoinselect}} amount')?></span>
```
Comment (DarrenWestwood) [inline order.php:37]: `{{altcoinselect}}` can be removed as there is no crypto selection.

File: order.php [hunk 3]
```diff
@@ -33,67 +25,52 @@
...
                   <span class="warning bnomics-status-warning" ng-show="order.status == -2" ng-cloak><?=__('Payment Error', 'edd-blockonomics')?></span>
                 </div>
-                    <h4 class="bnomics-amount-title" for="invoice-amount" ng-hide="order.status == -3">
-                     {{order.satoshi/1.0e8}} BTC
-                    </h4>
-                    <div class="bnomics-amount-wrapper" ng-hide="order.status == -3">
-                      <hr class="bnomics-amount-seperator"> ≈
-                      <span ng-cloak>{{order.value}}</span>
-                      <small ng-cloak>{{order.currency}}</small>
+                    <div>
```
Comment (DarrenWestwood) [inline order.php:41]: The checkout fields do not hide when the payment expires
EDD Payment expired

File: order.php [hunk 4]
```diff
@@ -33,67 +25,52 @@
...
                     </div>
               <!-- Bitcoin Address -->
-                <div class="bnomics-address" ng-hide="order.status == -3">
-                  <input ng-click="btc_address_click()" id="bnomics-address-input" class="bnomics-address-input" type="text" ng-value="order.address" readonly="readonly">
-                  <i ng-click="btc_address_click()" class="material-icons bnomics-copy-icon">file_copy</i>
-                </div>
-                <div class="bnomics-copy-text" ng-hide="order.status == -3 || copyshow == false" ng-cloak>Copied to clipboard</div>
+              <div class="bnomics-address">
+                    <div class="bnomics-address-text" ng-hide="address_copyshow" ng-cloak><?=__('To this bitcoin{{crypto.name | lowercase}} address', 'edd-blockonomics')?></div>
+                    <div class="bnomics-copy-address-text" ng-show="address_copyshow" ng-cloak><?=__('Copied to clipboard', 'edd-blockonomics')?></div>
```
Comment (DarrenWestwood) [inline order.php:53]: The copied to clipboard text does not display onclick. This js can be copied from the woocommerce plugin:
https://github.com/blockonomics/woocommerce-plugin/blob/master/js/app.js#L104-L156

File: order.php [hunk 5]
```diff
@@ -33,67 +25,52 @@
...
-                 </div>
-                 </br>
-                 <!-- Coin Select -->
-                 <div class="bnomics-address">
-                   <select ng-model="altcoinselect" ng-options="x for (x, y) in altcoins" ng-init="altcoinselect='Ethereum'"></select>
-                 </div>
-                 <div class="bnomics-altcoin-button-wrapper">
-                  <a ng-click="pay_altcoins()" href=""><button><i class="cf" ng-hide="altcoinselect!='Ethereum'" ng-class={'cf-eth':'{{altcoinselect}}'!=''} ></i><i class="cf" ng-hide="altcoinselect!='Litecoin'" ng-class={'cf-ltc':'{{altcoinselect}}'!=''} ></i> <?=__('Pay with', 'edd-blockonomics')?> {{altcoinselect}}</button></a>
-                 </div>
+                <div class="bnomics-powered-by">
```
Comment (DarrenWestwood) [inline order.php:67]: The Powered by Blockonomics section can be moved outside of `bnomics-btc-info` in order to remain centered on the page.
https://github.com/blockonomics/woocommerce-plugin/blob/master/templates/blockonomics_checkout.php#L99-L103

**Comment by DarrenWestwood** [comment]
Cryptofont can be removed as this no longer used:
https://github.com/blockonomics/easy-digital-downloads-plugin/blob/master/edd-blockonomics.php#L769-L770

**Comment by DarrenWestwood** [comment]
The code changes look good and do not break any functionality. 
When working with EDD, code can easily be reused from the woocommerce plugin.

File: order.php (hunk 3 above)
Comment (cnohall) [reply order.php]: Fixed

**Review by cnohall — COMMENTED** [review]
Thanks for the feedback! I've made changes according to the comments.

File: order.php (hunk 4 above)
Comment (DarrenWestwood) [reply order.php:56]: Clicking on the bitcoin amount should copy the bitcoin amount to the clipboard. You can complete this using  https://github.com/blockonomics/woocommerce-plugin/blob/master/js/app.js#L149-L152 and https://github.com/blockonomics/woocommerce-plugin/blob/master/templates/blockonomics_checkout.php#L74

**Comment by DarrenWestwood** [comment]
/css/cryptofont, /css/fonts, and /css/icons can be removed as well as the references to these files here: https://github.com/blockonomics/easy-digital-downloads-plugin/blob/master/edd-blockonomics.php#L769-L770

There is also some remaining css rules that can be removed. `.bnomics-payment-option`, `bnomics-copy-text`, `.bnomics-copy-icon`, and `.bnomics-amount-seperator`

File: order.php (hunk 4 above)
Comment (cnohall) [reply order.php:54]: Done

**Comment by cnohall** [comment]
Removed the files mentioned and the css.

**Comment by DarrenWestwood** [comment]
The checkout is looking good. There still seems to be some remaining code from the Flyp integration, which was called via [ajax/js](https://github.com/blockonomics/easy-digital-downloads-plugin/pull/29/files#diff-32cdce28ac549728f6a28999f6d874aa4f29e6da1406c28c3134cd8d421a866aL262-L268):
https://github.com/cnohall/easy-digital-downloads-plugin/blob/master/edd-blockonomics.php#L61-L66,
https://github.com/cnohall/easy-digital-downloads-plugin/blob/master/edd-blockonomics.php#L138-L148,
https://github.com/cnohall/easy-digital-downloads-plugin/blob/master/edd-blockonomics.php#L423,
https://github.com/cnohall/easy-digital-downloads-plugin/blob/master/edd-blockonomics.php#L430-L435,
https://github.com/cnohall/easy-digital-downloads-plugin/blob/master/edd-blockonomics.php#L734-L760,
https://github.com/cnohall/easy-digital-downloads-plugin/blob/master/order.php#L83

**Review by DarrenWestwood — APPROVED** [review]

File: css/order.css [hunk 6]
```diff
@@ -69,29 +67,43 @@
...
-.bnomics-address input , .bnomics-address select {
-  font-weight: 600 !important;
-  margin-top: 5px !important;
-  margin-bottom: 10px !important;
-  width: 90% !important;
-  border: 1px solid #DDD; !important;
-  background-color: #fff !important;
-  box-shadow: none !important;
+.bnomics-amount{
+  margin-top: 10px;
```
Comment (DarrenWestwood) [inline css/order.css:81]: One small ui change, the top margin can be removed from bnomics-amount as it is shifting the alignment of the text to the qrcode.

Comment (cnohall) [reply css/order.css]: Removed the top margin from bnomics-amount.

**Review by cnohall — COMMENTED** [review]
Removed top margin from bnomics-amount.

**Pull request merged**